<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditCalc</class>
 <widget class="QDialog" name="CreditCalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border: 1px solid gray;
border-radius: 10px;
background-color: qlineargradient(x1: 1, y1: -, x2: 0, y2: -1,
                                      stop: 0 #708090, stop: 1 #708090);
</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_procent">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>240</y>
     <width>140</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>140</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color:   rgb(0,0,0);
font: 8pt &quot;MV Boli&quot;;
background-color: rgb(211, 211, 211);
border-radius: 10px;
font-size: 33px;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="maxLength">
    <number>255</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>350</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>350</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 8pt &quot;MV Boli&quot;;
    border-radius:  10px;
    background-color:  rgb(37, 37, 37);
    color:  rgb(255, 255, 255);
    font-size:  35px;
}</string>
   </property>
   <property name="text">
    <string>Type of payments</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>350</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>350</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 8pt &quot;MV Boli&quot;;
    border-radius:  10px;
    background-color:  rgb(37, 37, 37);
    color:  rgb(255, 255, 255);
    font-size:  35px;
}</string>
   </property>
   <property name="text">
    <string>Credit/loan amount</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>350</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>350</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 8pt &quot;MV Boli&quot;;
    border-radius:  10px;
    background-color:  rgb(37, 37, 37);
    color:  rgb(255, 255, 255);
    font-size:  35px;
}</string>
   </property>
   <property name="text">
    <string>Term</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>350</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>350</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 8pt &quot;MV Boli&quot;;
    border-radius:  10px;
    background-color:  rgb(37, 37, 37);
    color:  rgb(255, 255, 255);
    font-size:  35px;
}</string>
   </property>
   <property name="text">
    <string>Interest rate</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>30</y>
     <width>500</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 8pt &quot;MV Boli&quot;;
    border-radius:  10px;
    background-color:  rgb(37, 37, 37);
    color:  rgb(255, 255, 255);
    font-size:  45px;
}</string>
   </property>
   <property name="text">
    <string>Credit Calculator</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_time">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>190</y>
     <width>140</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>140</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color:   rgb(0,0,0);
font: 8pt &quot;MV Boli&quot;;
background-color: rgb(211, 211, 211);
border-radius: 10px;
font-size: 33px;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="maxLength">
    <number>255</number>
   </property>
   <property name="cursorPosition">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_full">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>140</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>140</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;MV Boli&quot;;
background-color: rgb(211, 211, 211);
border-radius: 10px;
font-size: 33px;
color:   rgb(0,0,0);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="maxLength">
    <number>255</number>
   </property>
   <property name="cursorPosition">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>240</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 8pt &quot;MV Boli&quot;;
    border-radius:  10px;
    background-color:  rgb(37, 37, 37);
    color:  rgb(255, 255, 255);
    font-size:  35px;
}</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>190</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 8pt &quot;MV Boli&quot;;
    border-radius:  10px;
    background-color:  rgb(37, 37, 37);
    color:  rgb(255, 255, 255);
    font-size:  35px;
}</string>
   </property>
   <property name="text">
    <string>month(s)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>490</y>
     <width>350</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>350</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 8pt &quot;MV Boli&quot;;
    border-radius:  10px;
    background-color:  rgb(37, 37, 37);
    color:  rgb(255, 255, 255);
    font-size:  35px;
}</string>
   </property>
   <property name="text">
    <string>Overpayment</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>440</y>
     <width>350</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>350</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 8pt &quot;MV Boli&quot;;
    border-radius:  10px;
    background-color:  rgb(37, 37, 37);
    color:  rgb(255, 255, 255);
    font-size:  35px;
}</string>
   </property>
   <property name="text">
    <string>Monthly payment</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>540</y>
     <width>350</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>350</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 8pt &quot;MV Boli&quot;;
    border-radius:  10px;
    background-color:  rgb(37, 37, 37);
    color:  rgb(255, 255, 255);
    font-size:  35px;
}</string>
   </property>
   <property name="text">
    <string>Total payout</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_more">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>490</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color:   rgb(0,0,0);
font: 8pt &quot;MV Boli&quot;;
background-color: rgb(211, 211, 211);
border-radius: 10px;
font-size: 33px;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="maxLength">
    <number>1000</number>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_res_month">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>440</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color:   rgb(0,0,0);
font: 8pt &quot;MV Boli&quot;;
background-color: rgb(211, 211, 211);
border-radius: 10px;
font-size: 33px;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="maxLength">
    <number>1000</number>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_all">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>540</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color:   rgb(0,0,0);
font: 8pt &quot;MV Boli&quot;;
background-color: rgb(211, 211, 211);
border-radius: 10px;
font-size: 33px;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="maxLength">
    <number>1000</number>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_an">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>290</y>
     <width>170</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>170</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>170</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton
 {
	font: 8pt &quot;MV Boli&quot;;
    border-radius:  10px;
    background-color:  rgb(37, 37, 37);
    color:  rgb(255, 255, 255);
    font-size:  35px;
}</string>
   </property>
   <property name="text">
    <string>Annuity</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_dif">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>290</y>
     <width>260</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>260</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>260</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton
 {
	font: 8pt &quot;MV Boli&quot;;
    border-radius:  10px;
    background-color:  rgb(37, 37, 37);
    color:  rgb(255, 255, 255);
    font-size:  35px;
}</string>
   </property>
   <property name="text">
    <string>Differentiated</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_eq">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>370</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	font: 8pt &quot;MV Boli&quot;;
    border-radius:  10px;
    background-color:  rgb(37, 37, 37);
    color:  rgb(255, 255, 255);
    font-size:  35px;
}

QPushButton:hover {
    background-color:  rgb(118, 118, 118);
}

QPushButton:pressed {
    background-color:  rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>140</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 8pt &quot;MV Boli&quot;;
    border-radius:  10px;
    background-color:  rgb(37, 37, 37);
    color:  rgb(255, 255, 255);
    font-size:  35px;
}</string>
   </property>
   <property name="text">
    <string>$</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>110</y>
     <width>350</width>
     <height>230</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>230</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>350</width>
     <height>230</height>
    </size>
   </property>
   <property name="locale">
    <locale language="English" country="Europe"/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
